pkg/apps/alert/alert.go

```diff
@@ -94,6 +95,23 @@ func (a *Alert) getCreater(version string) (Creater, error) {
 	return nil, fmt.Errorf("version %s is not supported", version)
 }
 
+func (a *Alert) ensureVersion(alt *alertapi.Alert) error {
+	// If the version is not specified then we set it to be the latest.
+	versions := a.Versions()
+	if len(alt.Spec.Version) == 0 {
+		sort.Sort(sort.Reverse(sort.StringSlice(versions)))
```

Comment:
No need to sort right?  Just get the last one from the list?

Reply:
@jdartigalongue Why did you sort the versions when determining the latest?  @rrati I think it's because the latest one is not guaranteed to be the first element in the list.